### DISCUSSION TITLE
Can you calibrate an adapted lens with a crop sensor?

### DISCUSSION THREAD

**Discussion by Agilulfo** [lensfun/lensfun#2109] [Q&A] [answered]
I have a couple of vintage film lenses (m42 mount):

- super-takumar 200mm f4
- helios-44m-4 (not the MC version)

I could try to create a calibration for these lenses but I only own a camera body with a crop sensor (micro 4/3).

does it matter if the calibration process is done with one sensor size or another?
will the calibrations be usable if somebody adapt the same lens to a body with a different sensor size?

I've only found [this](https://github.com/lensfun/lensfun/discussions/1765) discussion related to multiple calibrations for the same lens with different sensors. It seems to suggest that bigger sensors should produce slightly better corrections.
However it is not clear if this is valid for all the 3 types of calibrations (distortion, vignetting, aberration).

Can somebody knowledgeable on the topic clarify the relevance/irrelevance of the sensor size?

**Comment by Agilulfo** [comment]
I might have found a related discussion in [this Issue](https://github.com/lensfun/lensfun/issues/2117) where the author created a profile for a full frame lens adapted to an aps-c sensor. 
My understanding is that when using an adapter to mount a lens to a body that is not compatible with the lens:
1. only the distortion correction can be added to the database .
2. in order to use the correction for a lens when you use and adapter you need[ to use the adapter function](https://github.com/lensfun/lensfun/blob/master/docs/man/lensfun-add-adapter.1.rst).

It is still unclear if the distortion correction is still meaningful for others or not. 

@tuxfanx could you clarify my question? :bow:

**Reply by tuxfanx** [reply, marked as answer]
to your 1st point:
No, all 3 corrections could be added to the database. However, it is always better if the pictures are taken with a camera that matches the lens mount. In #2117 there was a known problem, which I also explained there.
to your 2nd point:
The use of the lensfun-add-adapter function depends on the OS you use and the workflow application. You can use it but you don't have to in every case. In the issue mentioned, for example, the Laowa lens would not be available for an aps-c X mount camera under darktable because of the lens mount. One option is to use the above function to display the lens for X mount cameras. The function is not required under RawTherapee; all lens corrections can be selected manually.

The distortion correction in #2117 is usefull for all aps-c cameras with the laowa lens mounted and also for cameras with a smaller sensor than aps-c. It is not recommendet to use the correction on ff cameras ([read here under crop factor](https://lensfun.github.io/manual/v0.3.2/el_lens.html)).

**Reply by Agilulfo** [reply]
Great. thank you for the clarification :bow: and for your work
